src/components/DepositsTable/HeadRow.tsx

```diff
@@ -0,0 +1,95 @@
+import styled from "@emotion/styled";
+
+import { Text } from "components/Text";
+
+import { COLORS } from "utils";
+
+export type HeaderCells = typeof headerCells;
+
+export type ColumnKey = keyof HeaderCells;
```

Comment:
Nit: we should remove the white spaces to make this more cohesive. 
```suggestion
import styled from "@emotion/styled";
import { Text } from "components/Text";
import { COLORS } from "utils";

export type HeaderCells = typeof headerCells;
export type ColumnKey = keyof HeaderCells;
```

Reply:
https://github.com/across-protocol/frontend-v2/pull/908/commits/fba48dc6b508defe324363d5550f12a896246bb6